classes/client.js

```diff
@@ -130,6 +130,7 @@ module.exports = class Komada extends Discord.Client {
     }));
     await this.configuration.initialize(this);
     this.ready = true;
+    this.emit("log", `Ready to serve ${this.guilds.size} guilds.`);
```

Comment:
Needs to be more fancy. 1000x fancier for Komada.

Failure to do so will be 1000 years in the slow lane, no appeal.